core/scss/utilities/functions/modular-scale/_get-modular-spacing-multiplier.scss

```diff
@@ -0,0 +1,20 @@
+@charset "UTF-8";
+
+//
+// get-modular-spacing-multiplier($step)
+//
+// Get a value out of the modular-spacing-multiplier map.
+// Returns the value of the $modular-spacing-multiplier variable at a step(integer) on the scale.
+// The default for $step is 0, which returns a multiplier value of 1.
+//
+// $step - Integers from -4 to 5 including 0
+//
+// Style guide: Functions.ModularScale.get-modular-spacing-multiplier
+//
+@function get-modular-spacing-multiplier($step: 0) {
+  @if ( $step >= -4 and $step <= 5 ) {
+    @return map_get($modular-spacing-multiplier, $step);
+  } @else {
```

Comment:
Don't need an else statement here as the `@return` won't allow the function to continue.